website/source/guides/secret-mgmt/static-secrets.html.md

```diff
@@ -0,0 +1,549 @@
+---
+layout: "guides"
+page_title: "Static Secrets - Guides"
+sidebar_current: "guides-secret-mgmt-static-secrets"
+description: |-
+  Vault supports generating new unseal keys as well as rotating the underlying
+  encryption keys. This guide covers rekeying and rotating Vault's encryption
+  keys.
+---
+
+# Static Secrets
+
+Vault can be used to store any secrets in a secure manner.  The secrets may be
+SSL certificates and keys for your organization's domain, credentials to connect
+to a corporate database server, etc. Storing such sensitive information in a
+plaintext is not desirable. This guide demonstrates the use case of Vault as a
+Secret Storage.
+
+
+## Reference Material
+
+- [Key/Value Secret Backend](/docs/secrets/kv/index.html)
+- [Key/Value Secret Backend API](/apikey/secret/kv/index.html)
+- [Client libraries](/apikey/libraries.html) for Vault API for commonly used languages
+
+## Estimated Time to Complete
+
+10 minutes
+
+## Personas
+
+The end-to-end scenario described in this guide involves two personas:
+
+- **`devops`** with privileged permissions to write secrets
+- **`apps`** reads the secrets from Vault
+
+## Challenge
+
+Consider the following situations:
+
+- Developers use a single admin account to access a third-party app
+  (e.g. Splunk) and anyone who knows the user ID and password can log in as an
+  admin
+- SSH keys to connect to remote machines are shared and stored as a plaintext
+- API keys to invoke external system APIs are stored as a plaintext
+- An app integrates with LDAP, and its configuration information is in a
+  plaintext
+
+Organizations often seek an uniform solution to store any sensitive information
```

Comment:
Organizations often seek an uniform workflow to securely store this sensitive information.